src/RequestProcessor.cs

```diff
@@ -282,7 +298,7 @@ private void ProcessInvocationRequestImpl(StreamingMessage request, AzFunctionIn
                 // Invoke the function and return a hashtable of out binding data
                 Hashtable results = functionInfo.Type == AzFunctionType.OrchestrationFunction
                     ? InvokeOrchestrationFunction(psManager, functionInfo, invocationRequest)
```

Comment:
Even though we do not support ```Durable Functions``` yet, we should add the parameter ```FunctionInvocationPerformanceStopwatch stopwatch``` to ```InvokeOrchestrationFunction``` for completeness.

Reply:
Ok.